src/lib/Makefile.in

```diff
@@ -96,4 +96,5 @@ install: $(NDPI_LIBS)
 	cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)$(PREFIX)$(libdir)/
 	cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)$(PREFIX)$(libdir)/
 	mkdir -p $(DESTDIR)$(PREFIX)$(includedir)
-	cp ../include/*.h $(DESTDIR)$(PREFIX)$(includedir)
+	#Avoid installing private header
+	find ../include/*.h ! -name ndpi_private.h -exec cp "{}" $(DESTDIR)$(PREFIX)$(includedir)/ \;
```

Comment:
As much as I agree with that: I am not sure if @lucaderi will like that approach. I guess the idea was to make some broken third party apps work by moving the private header from `src/lib` to `src/include` in the first place.

Reply:
I am not concerned about the installation honestly, but what I am concerned is not to share (or include) config files such as ndpi_config.h that when referenced by other applications can create issues as all the autoconf libraries generate similar #defines such as PACKAGE_XXXX